vars/doPromotion.groovy

```diff
@@ -54,7 +50,7 @@ def call(Map promotion = [:]) {
             echo "Unable to get and parse rocketdict file from the Config File Provider plugin. Using default values..."
         }
 
-        // Grab the credentials stored in Jenkins or K8s secret used for each  url
+        // Retrieves credentials from the configuration file or uses default values.
```

Comment:
```suggestion
        // Get credentials from the Jenkinks configuration file if present (called 'rocketdict') or otherwise uses default values based-on Jenkins credentials parameters: rocket-auth-credentials & rocket-auth-credentials-target.
```

Reply:
done